### DISCUSSION TITLE
Stereo Tool Basic license best settings

### DISCUSSION THREAD

**Discussion by thehijacker** [AzuraCast/AzuraCast#5546] [Q&A]
Hello,

I have hearing loss and wear hearing aid behind my ears. This helps me with listening and understanding speech to some level. I love my Azuracast radio station made from all the songs I enjoy listening, but there are not all the same quality. Some are better quality than others and some are not on same volume level. So when such songs comes up I need to lower/increase my player volume.

Was reading a lot about Stereo Tool and finaly managed to make it work. Also purchased the basic license that should cover "Automatic Gain Control". The main issue I have now is with the configuration. Due to my hearing loss I have difficulties detecting the sound quality changes. I used some publicly available but they all need full licenses and to be honest, some songs were still to loud or to quiet to play.

Does anyone use Stereo Tool with basic license also and could share his settings file (sts) that works best for him?

Thank you.

**Comment by bpglobalnetworks** [comment]
the best way to work it is with the separate software, there you must also place the license and test the sound according to a playlist of songs that give you the volume problem. From there generate the sts file (just saving the preset, not all the configuration)
